blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,46 @@
+# First 🌱 Knative Virtual Office Hour
+
+**Date: 2022-12-01**
+
+**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), [Lance Ball](https://twitter.com/lanceball), and Paul Schweigert**
+
+We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.
```

Comment:
Can we add a splash image, like https://unsplash.com/photos/Q0AM87PsYkE ?

Reply:
Maybe one of photoa from KubeCon with Knative in background form KnativeCon or other meeting?

Reply:
👍 

That's a great idea!